stdlib/LinearAlgebra/test/triangular.jl

```diff
@@ -509,6 +509,30 @@ Atu = UnitUpperTriangular([1 1 2; 0 1 2; 0 0 1])
 @test typeof(sqrt(Atu)[1,1]) <: Real
 @test typeof(sqrt(complex(Atu))[1,1]) <: Complex
 
+@testset "matrix square root quasi-triangular blockwise" begin
```

Comment:
Should F32 be tested as well in case?

Reply:
`Float32`? Yes, good point, `ComplexF32` as well.